internal/config/credential_status.go

```diff
@@ -36,6 +38,11 @@ func (r *DirectStatus) GetURL() string {
 	return strings.TrimSuffix(r.URL, "/")
 }
 
+// GetAgentURL returns the URL of the agent endpoint
+func (r *DirectStatus) GetAgentURL() string {
+	return fmt.Sprintf("%s/v1/agent", strings.TrimSuffix(r.URL, "/"))
```

Comment:
not sure if it is the right place

Direct status was about an endpoint, this is more like Iden3CommAgentStatus, but maybe it is ok.

Reply:
I've changed this. It was wrong here